packages/dashboard/src/DashboardLayout.tsx

```diff
@@ -228,6 +231,11 @@ export function DashboardLayout({
     }
   }, [dehydrateComponent, isItemDragging, lastConfig, layout, onLayoutChange]);
 
+  const throttledHandleLayoutStateChanged = useMemo(
+    () => throttle(handleLayoutStateChanged, STATE_CHANGE_DEBOUNCE_MS),
```

Comment:
This will need the trailing option I think so the final call is always run. Not sure if that's a default or how the specifics work, but make sure the final call is always executed.

Reply:
`trailing=true` is the default.

Reply:
Looks like the default is `leading=true` and `trailing=true`. So I think it should be fine? But might want to add a `flush` call on unmount just in case there's a pending update